plugins/rhs_plugin/rhs_plugin.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import SearchResultsHeader from 'components/search_results_header';
+import {BoardsTourTip, PlaybooksTourTip} from 'components/tours/worktemplate_explore_tour';
+
+import Pluggable from 'plugins/pluggable';
+
+import AutoShowLinkedBoardTourTip from './auto_show_linked_board_tourtip';
+
+export type Props = {
+    showPluggable: boolean;
+    pluggableId: string;
+    title: string | React.ReactNode;
```

Comment:
Nit: Should be able to omit `string` since it is included in `React.ReactNode;`

Reply:
thanks, done!